src/AutoRest.CSharp/Common/Generation/Writers/LongRunningOperationWriter.cs

```diff
@@ -18,7 +16,7 @@ namespace AutoRest.CSharp.Generation.Writers
 {
     internal class LongRunningOperationWriter
     {
-        public static void Write(CodeWriter writer, LongRunningOperation operation)
+        public void Write(CodeWriter writer, LongRunningOperation operation)
```

Comment:
Refactor the common LongRunningOperationWriter so it can be better customized in the manangement plane child class